es/index.js

```diff
@@ -5,6 +5,9 @@ import {
   stringifyRequest,
   stringifyResponse,
 } from '@softonic/http-log-format';
+import pkg from '../package.json';
+
+const namespace = pkg.name;
```

Comment:
I used the name of the package as namespace to store information in the axios request configuration, so it doesn't conflict with other packages or axios itself.